core/client/hammer/hammer.go

```diff
@@ -76,125 +97,199 @@ func New(ctx context.Context, dial DialFunc, callOptions CallOptions,
 		timeout:     timeout,
 		ktCli:       ktCli,
 		appID:       fmt.Sprintf("hammer_%v", time.Now().Format("2006-01-02/15:04:05")),
-		config:      config,
+		domain:      domain,
 
 		signers:        []*tink.KeysetHandle{keyset},
 		authorizedKeys: authorizedKeys.Keyset(),
 	}, nil
 }
 
-// Run runs operationCount with up to maxWorkers over ramp duration.
-func (h *Hammer) Run(ctx context.Context, operationCount, maxWorkers int, ramp time.Duration) {
-	jobs := gen(ctx, operationCount)
-	outputs := h.launchWorkers(ctx, jobs, maxWorkers, ramp)
-	h.printStats(1*time.Second, outputs)
-}
+// Run runs a total of operationCount operations across numWorkers.
+// The number of workers should roughly be (goal QPS / timeout seconds)
```

Comment:
It seems reasonable to replicate this "number of workers" remark in the corresponding `workers` flag description.

Reply:
done